src/components/Books.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import { removeBook } from '../redux/books/books';
+import './Books.css';
+import 'react-circular-progressbar/dist/styles.css';
+
+const Books = ({ book }) => {
+  const [id, [item]] = book;
+  const dispatch = useDispatch();
+  const percentage = 66;
+
+  return (
+    <div className="books-contents">
+      <div className="first">
+        <li key={id} className="book-cont">
+          <span className="category">{item.category}</span>
+          <span className="title">{item.title}</span>
+          <span className="author">{item.author}</span>
+        </li>
+        <div className="buttons">
+          <button type="button" className="btn border"> Comment</button>
+          <button type="button" className="btn border" id={id} onClick={() => dispatch(removeBook(id))}>Remove</button>
+          <button type="button" className="btn">Edit</button>
+        </div>
+      </div>
+      <div className="prgs">
+        <div className="circular-progressbar" style={{ width: 100, height: 100 }}>
```

Comment:
- [ ] Kindly consider removing the inline styles if you want to, you can create a variable with the styles and then pass it as inline styles, this will make your code more organized

```
const myComponentStyle = {
  width: 100
  height: 100
}

    return (...
            <div className="circular-progressbar" style={myComponentStyle}>

```